src/components/atoms/LinkButton/LinkButton.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { DetailedHTMLProps, AnchorHTMLAttributes } from "react";
```

Comment:
Why are we creating a new `LinkButton` component from scratch here, when seemingly we already have `src/components/atoms/Button/Button.tsx` that has an `isLink` prop on it?

(cc // @mike-lvov FYI/thoughts?)

Reply:
The Button/Button.tsx uses react-router <Link />, which seemingly can't handle an navigating to an external URL. It probably should be renamed somehow, but I can see value in a component like it.

Reply:
I think that we could improve that later.

Reply:
Then we should create a backlog issue to track that.

Reply:
What kind of backlog issue should be created?